### PR TITLE
add timestamp string to console log

### PR DESCRIPTION
**Pull request by DanboDuan**
Signed-off-by: bob <bob170731@gmail.com>

#### What it does

add timestamp string to console log

#### How to test

see the backend or frontend console log, it should be like:
```
5/15/2022, 08:17:56.269 root INFO xxxx
```

#### Review checklist

- [x] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Comment by DanboDuan** [comment]
could you please help merging this pr @msujew ? This is also for logging output. Timestamp information should be helpful when tracing some important performance issue.

File: packages/core/src/common/logger-protocol.ts [hunk 1]
```diff
@@ -109,6 +109,8 @@ export namespace ConsoleLogger {
     export function log(name: string, logLevel: number, message: string, params: any[]): void {
         const console = consoles.get(logLevel) || originalConsoleLog;
         const severity = (LogLevel.strings.get(logLevel) || 'unknown').toUpperCase();
-        console(`${name} ${severity} ${message}`, ...params);
+        const now = new Date();
+        const timestamp = `${now.toLocaleString(undefined, { hour12: false })}.${now.getMilliseconds()}`;
```
Comment (msujew) [inline packages/core/src/common/logger-protocol.ts:113]: Leaving `locale` at undefined will lead to unexpected behavior on different systems. `en-GB` seems to be the most sensible, as it uses `day/month/year` with a 24-hour cycle (no `AM/PM`) by default.

```suggestion
        const timestamp = `${now.toLocaleString('en-GB')}.${now.getMilliseconds()}`;
```

**Review by msujew — COMMENTED** [review]
Works well, but I'm wondering whether we should use a better machine readable format such as `toISOString()`. Using ISO time would also be better in case the logs get sorted for some reason. @vince-fugnitto WDYT?

**Comment by vince-fugnitto** [comment]
> using ISO time would also be better in case the logs get sorted for some reason. @vince-fugnitto WDYT?

@msujew sounds good to me, we can use `toISOString()`.

**Comment by DanboDuan** [comment]
> > using ISO time would also be better in case the logs get sorted for some reason. @vince-fugnitto WDYT?
> 
> @msujew sounds good to me, we can use `toISOString()`.

OK, updated to use `toISOString()`

**Comment by DanboDuan** [comment]
@msujew @vince-fugnitto so could you please help merge this ?

**Pull request merged**